crates/contracts/solidity/Trader.sol

```diff
@@ -104,8 +105,12 @@ contract Trader {
             // We first reset the allowance to 0 since some ERC20 tokens (e.g. USDT)
             // require that due to this attack:
             // https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729
-            IERC20(sellToken).safeApprove(address(settlementContract.vaultRelayer()), 0);
-            IERC20(sellToken).safeApprove(address(settlementContract.vaultRelayer()), type(uint256).max);
+            try IERC20(sellToken).approve(address(settlementContract.vaultRelayer()), 0) {}
+            catch {}
+            try IERC20(sellToken).approve(address(settlementContract.vaultRelayer()), type(uint256).max) {}
+            catch {}
```

Comment:
nit: might be helpful to mention somewhere a reason why these errors can be ignored. The same applies to the deposit failures.

Reply:
Will add a comment for this but the `deposit` already has a comment [here](https://github.com/cowprotocol/services/pull/3160/files#diff-4aa907c4f7849db97dc730af3449827f5166e5da79f7a7f62c0647d1d097fb13R91-R92)